packages/manager/modules/vrack/src/dashboard/vrack.html

```diff
@@ -179,7 +179,7 @@ <h3 data-translate="vrack_description_title"></h3>
                 </p>
             </div>
         </section>
-
+        <!---->
```

Comment:
I imagine that the comment should include the name of the section

Reply:
yup, finally it is cleaner without comment